.kokoro/tests/run_tests.sh

```diff
@@ -50,6 +50,7 @@ gcloud auth activate-service-account \
 
 export GOOGLE_CLIENT_SECRETS=$(pwd)/testing/client-secrets.json
 source "${KOKORO_GFILE_DIR}/automl_secrets.txt"
+source "${KOKORO_GFILE_DIR}/talent_sln_secrets.txt"
```

Comment:
What's inside? Is it really a secret?

Reply:
It seems like just few ids. They don't seem like real secrets.

Reply:
I named after other text file names. What name should I rename to?